_posts/2019-11-24-foreman-community-newsletter---november-2019.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: Foreman Community Newsletter (November 2019)
+date: 2019-11-24 13:46:53
+author:
+ name: Ori Rabin
+ uri: orabin
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+ We're back! CFP still open for Fosdem and CfgMgmtCamp and lots of releases!
+---
+
+Welcome back to the Foreman Newsletter! After a few months full of conferences 
+that kept our community team very busy, we are now back for good :) 
+
+
+### 1.24 RC2 && Katello 3.14 RC1 are out!
```

Comment:
unless this goes out today we'll be at RC3 already

Reply:
I'd be glad if this goes out today/tomorrow to make sure people see the CFP dates before they end.